src/jasmin/sha.rs

```diff
@@ -22,17 +21,15 @@ pub fn hash<const LEN: usize>(alg: Algorithm, payload: &[u8]) -> Result<[u8; LEN
         Algorithm::Blake2b => todo!(),
         Algorithm::Sha3_224 => {
             if simd256_support() {
-                log::trace!("Jasmin SHA3 avx2");
-                unsafe {
+                                unsafe {
                     libjade::jade_hash_sha3_224_amd64_avx2(
                         digest.as_mut_ptr(),
                         payload.as_ptr() as _,
                         payload.len().try_into().unwrap(),
                     )
                 }
             } else {
-                log::trace!("Jasmin SHA3 ref");
-                unsafe {
+                                unsafe {
```

Comment:
Also same question; was this automatically formatted by cargo? Running `cargo fmt` on the dev branch does not give me this diff.

Reply:
Ugh, that's weird. Looks like I didn't fmt after removing the logging.